recipes-omnect/initrdscripts/omnect-os-initramfs/omnect-device-service-setup

```diff
@@ -80,6 +82,12 @@ run_omnect_device_service_setup () {
         # a possible reason to not do it here could be that we never
         # reach aziot-identityd-precondition
     fi
+    if [ -n "${omnect_bootloader_updated_flag}" ]; then
+        run_cmd touch ${omnect_bootloader_updated_file} \
+        && run_cmd chown ${omnect_device_service_user}:${omnect_validate_update_group} ${omnect_bootloader_updated_file} \
```

Comment:
`omnect_validate_update_group` is an empty string

Reply:
done